src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -341,11 +353,41 @@
     (->> (map format-task info-from-jruby)
          (middleware-utils/json-response 200))))
 
+(schema/defn ^:always-validate
+  describe-task :- TaskDetails
+  [task-data :- TaskData
+   env-name :- schema/Str
+   module-name :- schema/Str]
+  (let [?metadata (try (some-> task-data :metadata_file slurp json/decode)
+                   (catch Exception e
+                     nil))
```

Comment:
Hrm, not returning anything about the exception seems less than ideal. Can we at least log it?

Reply:
yeah, logging it is a good idea